utils/grpc/grpc.go

```diff
@@ -44,11 +47,104 @@ func X509Certificate(privKey crypto.PrivateKey) (*tls.Certificate, error) {
 	return &libp2pTlsConfig.Certificates[0], nil
 }
 
-// DefaultServerTLSConfig returns the default TLS config with the given cert for a secure GRPC server
+// DefaultServerTLSConfig returns the default TLS server config with the given cert for a secure GRPC server
```

Comment:
Please review https://github.com/onflow/flow-go/pull/990 and add a MinTLSVersion >= 1.2

Reply:
Added `MinVersion: tls.VersionTLS13,` to ServerTLSConfig